charts/console/templates/tests/02-credentials-secret.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and (eq .Values.test true) .Values.config.existingSecret }}
+{{- $secretName := include "conduktor.platform.secretName" . }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $secretName }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: conduktor-platform
+type: Opaque
+stringData:
+  CDK_ORGANIZATION_NAME: {{ .Values.config.organization.name | quote }}
+  CDK_ADMIN_EMAIL: {{ .Values.config.admin.email | quote }}
+  CDK_ADMIN_PASSWORD: {{ .Values.config.admin.password | quote }}
+  CDK_DATABASE_PASSWORD: {{ .Values.config.database.password | quote }}
+  CDK_DATABASE_USERNAME: {{ .Values.config.database.username | quote }}
```

Comment:
Missing base64 encode for all secrets, you can replace `quote` with `b64enc | quote`

Reply:
As we're using stringData, no need for b64

Reply:
I missed that thanks